pkg/compiler/codegen.go

```diff
@@ -267,6 +267,9 @@ func (c *codegen) emitLoadVar(pkg string, name string) {
 	vi := c.getVarIndex(pkg, name)
 	if vi.ctx != nil && c.typeAndValueOf(vi.ctx.expr).Value != nil {
 		c.emitLoadConst(c.typeAndValueOf(vi.ctx.expr))
+		if vi.ctx.assertToType != nil {
+			c.emitConvert(*vi.ctx.assertToType)
```

Comment:
`emitLoadVar` is too generic place to be doing this to me.

Reply:
How about
```
func Notify(name string, args ...interface{}) {
        neogointernal.Syscall2NoReturn("System.Runtime.Notify", name, neogointernal.CastNotificationArgs(args))
}
```
and handle it in `neogointernal.CastNotificationArgs` code generator?

Reply:
This code wasn't correct at all. It was amed to handle ellipsis case, but in fact it made a mess. I've removed it completely.

So currently we're able to emit arguments conversion code for `runtime.Notify("name", arg1, arg2, arg3)` and still not able to emit conversion code for ellipsis expressions like `runtime.Notify("name", arr...)`. Ellipsis are complicated case, because we need to emit code that traverses given array by ourselves, and I'm not sure how to implement this.

Reply:
OK, I've just got the idea, instead of direct usage of `neogointernal.CastNotificationArgs` we can write something like
```
func Notify(name string, args ...interface{}) {
	for i := range args {
		neogointernal.CastNotificationArg(args[i])
	}
	neogointernal.Syscall2NoReturn("System.Runtime.Notify", name, args)
}
```
This will do the traversing job for us, ao the only thing left is the cast itself, I'll try it.

Reply:
Bad idea, it won't work this way. We decided to support argument types cast for non-ellipsis only.